js/directives/day-block.js

```diff
@@ -0,0 +1,63 @@
+/* global angular */
+(function(){
+    'use strict';
+    var DayBlockDirective = function () {
+        return {
+            template : 
+            '<div class="day-bloc">'+
```

Comment:
that's not a typo, the css class day-bloc exists (maybe the original typo was into the scss)